src/state-incentive-details.tsx

```diff
@@ -360,29 +204,93 @@ const isIRARebate = (incentive: Incentive) =>
 /** TODO get real dates in the data! */
 const renderStartDate = (incentive: Incentive) =>
   isIRARebate(incentive) ? (
-    <div className="incentive__chip incentive__chip--warning">
-      <ExclamationPoint w={16} h={16} /> {msg('Expected in 2024')}
-    </div>
+    <Chip isWarning={true}>{msg('Expected in 2024')}</Chip>
   ) : null;
 
+const Chip: FC<PropsWithChildren<{ isWarning?: boolean }>> = ({
+  isWarning,
+  children,
+}) => {
+  const classes = [
+    'flex',
+    'w-fit',
+    'gap-2.5',
+    'items-center',
+    'justify-center',
+    'rounded',
+    'font-bold',
+    'text-xsm',
+    'leading-tight',
+    'tracking-[0.03438rem]',
+    'uppercase',
+  ];
+  if (isWarning) {
+    classes.push(
+      'bg-yellow-200',
+      'text-red-500', // TODO should be #806c23
```

Comment:
This color doesn't seem to be in the design system; I'm not sure what to do about that. This text was originally spec'd as `#846f24`, which doesn't seem to be in the design system either (?) and does not have enough contrast with bg-yellow-200. I just put red-500 here as a placeholder, but it doesn't look good.

Reply:
For parity with the old code you can specify arbitrary one-offs in tailwind like `bg-[#806c23]` and then we can follow up to see what should happen design-system wise.

Reply:
I see you already did that with the tracking value - I think that's fine for now.